src/components/JobConfiguration.vue

```diff
@@ -235,7 +239,11 @@ export default defineComponent({
       if (job?.statusId === 'SERVICE_DRAFT') {
         this.store.dispatch('job/scheduleService', job)
       } else if (job?.statusId === 'SERVICE_PENDING') {
-        this.store.dispatch('job/updateJob', job)
+        this.store.dispatch('job/updateJob', job).then( async (resp) => {
+          if(resp) {
+            await this.store.dispatch('job/fetchPendingJobs', { eComStoreId: this.getCurrentEComStore.productStoreId, viewSize: process.env.VUE_APP_VIEW_SIZE, viewIndex: 0, jobEnums: this.jobEnums });
```

Comment:
I think this will dispatch even when user is updating job from page other than pipeline page. This should be properly handled.